src/sources/socket/mod.rs

```diff
@@ -899,12 +899,27 @@ mod test {
 
     //////// UDP TESTS ////////
     fn send_lines_udp(addr: SocketAddr, lines: impl IntoIterator<Item = String>) -> SocketAddr {
-        send_packets_udp(addr, lines.into_iter().map(|line| line.into()))
+        send_lines_udp_from(next_addr(), addr, lines)
+    }
+
+    fn send_lines_udp_from(
+        from: SocketAddr,
+        addr: SocketAddr,
```

Comment:
Nit: 
```suggestion
        to: SocketAddr,
```

Here and bellow

Reply:
Nice catch!